src/decorators/swagger/api-common-error-code.swagger.ts

```diff
@@ -0,0 +1,19 @@
+import { HttpStatus, applyDecorators } from '@nestjs/common';
+import { ApiBearerAuth } from '@nestjs/swagger';
+import { COMMON_ERROR_CODE } from '@src/constants/error/common/common-error-code.constant';
+import { HttpException } from '@src/http-exceptions/exceptions/http.exception';
+
+export function ApiCommonErrorCode(): ClassDecorator {
```

Comment:
```suggestion
export function ApiCommonErrorCode() {
```
method 에만 쓸 수 있는 경우도 있으니 타입을 빼고 applyDecorator가 반환해주는 리턴타입을 그대로 쓰는게 나을거같아요.
ClassDecorator | MethodDecorator 처럼 쓸 수도 있지만 그러면 호출부에서 타입이 깨집니다.